web/public/js/directives/cd-start-dojo/champion/index.js

```diff
@@ -70,6 +70,22 @@ angular
           ctrl.getAge();
           ctrl.formatDoB();
         }, true);
+
+        var phoneWatcher = $scope.$watch('$ctrl.champion.phone', function () {
+          if (ctrl.champion && ctrl.champion.phone) {
+            $timeout(function () {
+              // Force reload of value with async loading of data
+              var element = $('input[name="phone"]');
```

Comment:
Could inject `$element` and use that to find the input to scope it to the directive.